clink_dev/README.md

```diff
@@ -0,0 +1,46 @@
+## Clink Pipeline
+
+Clink Pipeline是基于Alink的特征工程pipeline构建工具，用于生成libfg转换特征使用的配置文件。
+
+Clink Pipeline的工作流程如下：
+
+1. 编写任务算子配置文件，配置一次任务中使用到的算子和算子对应的参数。
+2. 编写待处理数据的schema文件。
+3. 使用Clink Pipeline读取上述配置文件和任务数据，进行fit操作，得到libfg transform所需的配置文件。
+4. 打包压缩产出的配置文件，上传至指定的S3集群。
+5. libfg加载S3集群中的配置文件，提供特征转换服务。
+
+##### 上述操作支持本地和VVP平台两种运行方式，数据读取支持本地数据和HDFS数据。
```

Comment:
Remove the usage of the VVP platform, because the VVP platform is the Aliyun internal platform.